app/workers/publishing_api_corporate_information_pages_worker.rb

```diff
@@ -0,0 +1,39 @@
+class PublishingApiCorporateInformationPagesWorker
+  extend Forwardable
+  include Sidekiq::Worker
+
+  def perform(edition_id, event)
```

Comment:
Do we need to pass the event in here? If all events end up aliased to `publish` it just seems to add complexity to 'handle' all of the events.